tests/test_advisory_grade.py

```diff
@@ -4,6 +4,7 @@
 from oar.core.configstore import ConfigStore
 from oar.core.exceptions import AdvisoryException
 
+
```

Comment:
Why there is one more empty line?

Reply:
It's a common code style in python. See https://peps.python.org/pep-0008/#blank-lines